installer/win/Brackets_fr-fr.wxl

```diff
@@ -53,7 +53,7 @@
     <String Id="ExitDialogTitle" Overridable="yes">{\WixUI_Font_BannerTitle}Procédure de l’assistant d’installation de [ProductName] terminée</String>
     <String Id="VerifyReadyDlgInstallText" Overridable="yes">Cliquez sur Installer pour lancer la procédure ou sur Annuler pour quitter.</String>
     <String Id="VerifyReadyDlgRemoveText" Overridable="yes">Cliquez sur Supprimer pour désinstaller !(loc.ShortProductName) ou sur Annuler pour quitter.</String>
-    
+    <String Id="OpenWithText" Overridable="yes">Open With !(loc.ShortProductName)</String>
```

Comment:
We only have two language installers. And this is just a placeholder before our LOC team subsitutes with french translation.